src/main.go

```diff
@@ -9,18 +8,9 @@ import (
 	"time"
 
 	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/managers"
-	"github.com/joho/godotenv"
 )
 
 func main() {
-	log.Println("Loading environment variables...")
-	err := godotenv.Load()
```

Comment:
Why did you delete this?

Reply:
I'm not sure if we should include the .env file in the Docker image. Therefore, I removed the 'godotenv' library. The new approach to running it would be to execute the 'set-and-run.sh' script; within that script, we set the environment variables using the 'export' command.